test/puppetlabs/puppetdb/fixtures.clj

```diff
@@ -106,6 +106,12 @@
                      (java.util.UUID/randomUUID)
                      ";hsqldb.tx=mvcc;sql.syntax_pgs=true")})
 
+(defn skip-hsqldb
```

Comment:
I think what you really want here is a test selector. We use the something similar for the opposite case [here] (https://github.com/puppetlabs/puppetdb/blob/master/test/puppetlabs/puppetdb/http/facts_test.clj#L577)